lib/ui/navbar.js

```diff
@@ -42,9 +43,47 @@ export default class IpNavBar extends React.Component {
     this.onDisconnect = this.onDisconnect.bind(this);
     this.toggleHelpVisible = this.toggleHelpVisible.bind(this);
     this.toggleColorScheme = this.toggleColorScheme.bind(this);
+    this.hideNavBarTimer = null;
+    this.hideIpNavBar = this.hideIpNavBar.bind(this);
+    this.showIpNavBar = this.showIpNavBar.bind(this);
+    this.renderVisible = this.renderVisible.bind(this);
+    this.renderHide = this.renderHide.bind(this);
+  }
+
+  hideIpNavBar() {
+    if (
+      this.props.model.get().settingsVisible ||
+      !this.props.model.get().isLocked ||
+      this.props.model.get().helpVisible ||
+      !this.state.connected) return;
+    this.setState({visible: false});
+  }
+
+  showIpNavBar(evt) {
+    clearInterval(this.hideNavBarTimer);
```

Comment:
Shouldn't this be `clearTimeout`, if for no other reason than clarity? And the other instance as well.

https://stackoverflow.com/questions/9913719/are-cleartimeout-and-clearinterval-the-same

Reply:
Absolutely no reason, just a syntax error. It's fixed.